docker/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN bash -c 'if test -n "$http_proxy"; then git config --global http.proxy "$htt
 RUN bash -c 'if test -n "$https_proxy"; then git config --global https.proxy "$https_proxy"; fi'
 RUN bash -c 'if test -n "$no_proxy"; then git config --global core.noproxy "$no_proxy"; fi'
 
-RUN pip install --user cyclonedx-python-lib==7.3.1 --break-system-packages
+RUN pip install --user cyclonedx-python-lib==7.5.1 --break-system-packages
```

Comment:
How about `pip install 'cyclonedx-python-lib>=7.3.2,<=7.5.1'` ?
Specific version may be yanked in the future.

Reply:
@hiraku-wfs thanks. I'll apply this command line.

Reply:
@hiraku-wfs I pushed new commit.